src/puppetlabs/puppetdb/client.clj

```diff
@@ -14,28 +14,28 @@
 
 (defn-validated submit-command-via-http!
   "Submits `payload` as a valid command of type `command` and
-  `version` to the PuppetDB instance specified by `host` and
-  `port`. The `payload` will be converted to JSON before
-  submission. Alternately accepts a command-map object (such as those
-  returned by `parse-command`). Returns the server response."
+   `version` to the PuppetDB instance specified by `host` and
+   `port`. The `payload` will be converted to JSON before
+   submission. Alternately accepts a command-map object (such as those
+   returned by `parse-command`). Returns the server response."
   ([base-url
     command :- s/Str
     version :- s/Int
     payload]
-     (->> payload
-          (command/assemble-command command version)
-          (submit-command-via-http! base-url)))
+   (->> payload
+        (command/assemble-command command version)
+        (submit-command-via-http! base-url)))
   ([base-url :- utils/base-url-schema
     command-map :- {s/Any s/Any}]
-     (let [message (json/generate-string command-map)
-           checksum (kitchensink/utf8-string->sha1 message)
-           url (str (utils/base-url->str base-url)
-                    (format "/commands?checksum=%s" checksum))]
-       (http-client/post url {:body               message
-                              :throw-exceptions   false
-                              :content-type       :json
-                              :character-encoding "UTF-8"
-                              :accept             :json}))))
+   (let [message (json/generate-string command-map)
+         checksum (kitchensink/utf8-string->sha1 message)
+         url (str (utils/base-url->str base-url)
+                  (format "/commands?checksum=%s" checksum))]
+     (http-client/post url {:body               message
+                            :throw-exceptions   false
+                            :content-type       :json
+                            :character-encoding "UTF-8"
+                            :accept             :json}))))
```

Comment:
(...and I say that, even though some of the really long lines further down definitely make my reformatting finger twitch.)